src/vmm_trie.ml

```diff
@@ -1,10 +1,11 @@
 (* (c) 2018 Hannes Mehnert, all rights reserved *)
 
+module Map = Map.Make(Vmm_core.Name.Label)
 (* each node may have a value (of type 'a), the boolean represents whether it
    is a path or a name (i.e. foo:bar: <value> or foo:bar <value>). *)
-type 'a t = N of ('a * bool) option * 'a t Vmm_core.String_map.t
```

Comment:
todo: check whether the String_map is still used somewhere, otherwise remove it from Vmm_core

Reply:
And thanks for looking into this :)